mkosi

```diff
@@ -56,6 +56,14 @@ class OutputFormat(Enum):
     directory = 4
     subvolume = 5
     tar = 6
+    raw_xfs = 7
+
+RAW_FORMATS = [
+  OutputFormat.raw_gpt,
```

Comment:
hmm, if we add this i think it would make sense to also rename raw_gpt to raw_ext4 then, otherwise it gets too confusing...(maybe in a separate commit either before or after this one) there should be some compat however, so that when the user uses the old name we do the right thing still, even tough we'd only advertise and document the new name

Reply:
I was thinking this too, but wasn't sure if ext4 on `raw_gpt` had some special meaning for a specific purpose.

Have renamed everything to `raw_ext4`, including the command help, though `raw_gpt` is still in
the enum with the same value, so shouldn't break anything for anyone.